library/src/morpheus-master/sass/base/_bootstrap-defaults.scss

```diff
@@ -210,7 +210,8 @@ $icon-font-path: if($bootstrap-sass-asset-helper, "bootstrap/", "../" + $skin-na
 //** `<input>` background color
 // $input-bg:                       #fff
 //** `<input disabled>` background color
-// $input-bg-disabled:              $gray-lighter
+//Chrome default
+$input-bg-disabled:               rgb(235, 235, 228);
```

Comment:
why not just redefine gray-lighter?

Reply:
Okay, I can do that too. It looks like the bootstrap coded default is 
`$gray-lighter:           lighten($gray-base, 46.7%) !default; // #eee`

Which seems like it should be "gray-light"
` lighten($gray-base, 93.5%) `

looks okay too

Reply:
Updated PR